### ISSUE TITLE
Convert all .rst files to the .md format

### ISSUE THREAD

**Issue by bocharsky-bw** [knpuniversity/oauth#59]
According to [this](https://github.com/knpuniversity/twig/pull/1#issuecomment-280525836) comment

**Comment by bocharsky-bw** [comment]
Hey @larzuk91 ,

I'd really like your help with this issue! I did it for Twig tut, here's an example what you need to do:
https://github.com/knpuniversity/twig/pull/2

As Ryan said, it's not urgent. If you have any questions - just ask! ;)

**Comment by larzuk91** [comment]
Ohh, for sure! Let me check some things because I haven't done this before :)

**Comment by larzuk91** [comment]
I have a question, how can I check that I did it right ?
I'm using `KnpUMarkdown `to parse and get the html content, after that I go and edit a chapter, changing `script_content` field and check it on the web
or there is another way ? :)

**Comment by bocharsky-bw** [comment]
@larzuk91 No, we can't since we simplified course building system lately. We'll see it when push to master and KnpU sync that content, but before this, I can review your work when you push a PR ;)

**Comment by larzuk91** [comment]
Ohh interesting, so I'll be extra careful :)

**Comment by bocharsky-bw** [comment]
@larzuk91 It makes no big sense... if we miss something, I bet it will something minor which we'll see on KnpU can quickly fix it ;)
